game/script-farewells.rpy

```diff
@@ -345,17 +345,45 @@ label farewell_option_work:
         n 1kllajsbr "But...{w=0.5}{nw}" 
         extend 1knmgssbr " it's super [time_concern] though,{w=0.1} [player]..."
         n 1kllsll "..."
-        n 1fcspusbr "Just...{w=1}{nw}" 
-        extend 1kllsl " be careful,{w=0.2} alright?"
-        n 1fsqpol "And you {i}better{/i} come visit when you get back."
-
-        if Natsuki.isLove(higher=True):
-            n 1fnmcal "Take care,{w=0.1} [player]!{w=1}{nw}"
-            extend 1kchsmleaf " I love you!"
-            n 1kllcalsbr "..."
-
-        else:
-            n 1fnmcal "Take care,{w=0.1} [player]!"
+        n 1kllajl "Are you..."
+        show natsuki 1tnmbol
+
+        menu:
+            n "Are you at {i}least{/i} working from home today,{w=0.2} or...?"
+
+            "Yes, I'm working from home.":
+                n 1ncsssl "Heh.{w=0.75}{nw}"
+                extend 1nllpul " I {i}guess{/i} that's something,{w=0.75}{nw}"
+                extend 1nslsll " at least."
```

Comment:
I'm wondering if it comes off a bit uninspired with relying on the word "at least" twice or if you just wanted to highlight this expression

Reply:
Ah, good catch - I need to correct that